tests/RequirementsTest.py

```diff
@@ -0,0 +1,64 @@
+import os
+import sys
+import unittest
+
+from coalib.misc.Shell import run_shell_command
+
+
+class coalaaCITest(unittest.TestCase):
+
+    def test_find_no_issues(self):
+        with open(os.path.dirname(__file__) + '/../requirements.txt') as f:
+            content = f.readlines()
+        reqiredpac = []
+        req_pac_ver = []
+
+        for x in content:
+            if '=' in x:
+                reqiredpac.append(x[:x.find('=')-1])
+                req_pac_ver.append(x[x.find('=')+1:-1])
+
+        with open(os.path.dirname(__file__) + '/../test-requirements.txt') as f:
+            content = f.readlines()
+        for x in content:
+            if '=' in x:
+                reqiredpac.append(x[:x.find('=')-1])
+                req_pac_ver.append(x[x.find('=')+1:-1])
+
+        output = run_shell_command([sys.executable, '-m', 'pip', 'freeze'])
+        split_list = output[0].split('\n')
+        package = []
+        temp = []
+        for i in split_list:
+            package.append(i[:i.find('=')])
+            temp.append(i[i.find('==')+2:])
+
+        package = [x.lower() for x in package]
+        reqiredpac = [x.lower() for x in reqiredpac]
+        package = [x.replace('-', '_') for x in package]
+        reqiredpac = [x.replace('-', '_') for x in reqiredpac]
+
+        version = []
+        for k in temp:
+            while(k.count('.') > 1):
+                k = k[:k.rfind('.')]
+            version.append(k)
+
+        req_version = []
+        for k in req_pac_ver:
+            while(k.count('.') > 1):
+                k = k[:k.rfind('.')]
+            req_version.append(k)
```

Comment:
This may introduce a bug later I guess. 
I think this might be useful http://stackoverflow.com/questions/11887762/compare-version-strings